### PR TITLE
Communication chat test scenario remove participant - updated the error code in comment

### PR DESCRIPTION
**Pull request by v-vprasannak**
Communication chat test scenario remove participant - updated the error code in comment

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/communication/azext_communication/tests/latest/test_communication_chat_scenario.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ def test_chat_remove_participants(self, communication_resource_info):
             self.cmd('az communication chat participant remove --thread {thread_id} --user {user_id} --yes')
 
         # assert 'The initiator doesn\'t have the permission to perform the requested operation.' in str(raises.exception)
-        # For now rest endpoint returns a 500 error, so we are checking for that
```
Comment (kairu-ms) [inline src/communication/azext_communication/tests/latest/test_communication_chat_scenario.py:268]: It's not necessary to release a new version if only the test code changed.

Comment (v-vprasannak) [reply src/communication/azext_communication/tests/latest/test_communication_chat_scenario.py:268]: @kairu-ms, reverted the version upgrade changes

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**